microsoft-365/admin/add-users/give-mailbox-permissions-to-another-user.md

```diff
@@ -28,7 +28,7 @@ ms.assetid: 1dbcf12f-a9de-4d1d-b0b3-a227f8a736d8
 description: "Give a user the right to access another user's mailbox, which allows the user to read and send emails from the other user's mailbox."
 ---
 
-# Give mailbox permissions to another user - Admin Help
+# Give mailbox permissions to another user - Admin help
```

Comment:
@lhcole613 - Don't we need to include the product name in the heading? And is there a reason why we add "- Admin help" to this H1? We don't typically add that to H1 headings for other admin topics, do we?

Reply:
@SKjerland, we added these to the end of Docs topics that endusers were landing on and then getting confused. Not sure what we're doing with those topics now.

Reply:
Sounds like a bigger conversation, but I would like to add the product name in the H1, but placement here is interesting. Does this work? Give mailbox permissions to another Microsoft 365 user - Admin help

Reply:
@lhcole613 - That looks good to me, Leslie! Thanks!